vsn-check.sh

```diff
@@ -1,23 +0,0 @@
-#!/bin/bash -e
-
-PROJECT_VERSION="$1"
-SNAPPYER_VSN="$2"
-
-APP_VSN=$(erl -noshell -eval '{ok, [{_,_,L}]} = file:consult("src/kafka_protocol.app.src"), {vsn, Vsn} = lists:keyfind(vsn, 1, L), io:format("~s", [Vsn]), halt(0).')
```

Comment:
we used to have version defined in Makefile and .app.src
Now it's in .app.src only.